client/landing/gutenboarding/components/domain-categories/style.scss

```diff
@@ -1,5 +1,47 @@
 @import '../../mixins';
 
+.domain-categories {
+
+	@media ( max-width: $break-medium ) {
+		margin-bottom: 20px;
+
+		.domain-categories__dropdown-button.components-button {
+			display: block;
+		}
```

Comment:
The arrow seems a little too high, this helps but not entirely:

Before:
![image](https://user-images.githubusercontent.com/17054134/80969874-9945be00-8e1a-11ea-9cc8-def1cd5f1d3d.png)

After:
![image](https://user-images.githubusercontent.com/17054134/80969831-8337fd80-8e1a-11ea-861b-be3a20b0a983.png)
```suggestion
		.domain-categories__dropdown-button.components-button {
			display: block;
			
			> * {
				vertical-align: middle
			}
		}
```

Reply:
Done.